validphys2/src/validphys/comparefittemplates/report.md

```diff
@@ -11,6 +11,8 @@ We are comparing:
 
 {@ summarise_fits @}
 
+{@ dataspecs::t0_info t0_chi2_info_table @}
```

Comment:
We might want to have a heading here.

Reply:
Now it's under the heading 'summary' where I would say it should be, I think a subsection under summary would be excessive. Do you disagree?

Reply:
Fair enough. The entries themselves say that is the t0.

Reply:
Exactly, I just made them separate tables since they can rely on different t0pdfsets